scalding-core/src/main/scala/com/twitter/scalding/Job.scala

```diff
@@ -521,11 +521,11 @@ trait CounterVerification extends Job {
   }
 }
 
-private[scalding] object FlowStepStrategies {
+private[scalding] case class FlowStepStrategies[A]() extends Semigroup[FlowStepStrategy[A]] {
```

Comment:
I think strategies will be applied in a sequence so it is associative (and not commutative)? /cc @ianoc  as he wrote this.